pylint_odoo/checkers/modules_odoo.py

```diff
@@ -826,13 +827,14 @@ def _check_xml_attribute_translatable(self):
         """The xml attribute is missing the translation="off" tag
             Example  <attribute name="groups">sale.group</attribute>
         """
-        self.msg_args = []
-        for xml_file in self.filter_files_ext('xml', relpath=True):
-            for record in self.get_xml_records(
-                    os.path.join(self.module_path, xml_file), None,
-                    '//attribute[not(@translation)]'):
-                self.msg_args.append(
-                    ("%s:%d" % (xml_file, record.sourceline), 'xml_id'))
-        if self.msg_args:
-            return False
+        if ODOO_VERSION == '8.0':
```

Comment:
Add a new check too to check if the attribute of translation from xml exists for a version of odoo != 8.0 then emit a message of `redundant-xml-attribute-translatable`